src/software/thunderscope/gl/layers/gl_layer.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, name: str) -> None:
 
```

Comment:
Not sure whether `@abstractmethod` is needed here

Reply:
We actually removed `ABC` and `@abstractmethod` from a couple classes last saturday bc it was causing issues